assets/minecraft/lang/en_US.lang

```diff
@@ -15,3 +15,215 @@ item.appleGold.name=Golden Apple
 item.goldNugget.name=Glod Nugget
 item.carrotGolden.name=Golden Carrot
 item.horsearmorgold.name=Glod Horse Armor
+tile.ston.ston.name=Ston
```

Comment:
Crap. Yeah, I used the "replace" function of Notepad and told it to match case, because I thought all the ones before the equal sign were lower case.